lib/fluent/plugin/parser_json.rb

```diff
@@ -52,9 +52,9 @@ def configure_json_parser(name)
         when :oj
           return [Oj.method(:load), Oj::ParseError] if Fluent::OjOptions.available?
 
-          log&.info "Oj is not installed, and failing back to Yajl for json parser"
-          configure_json_parser(:yajl)
-        when :json then [JSON.method(:load), JSON::ParserError]
+          log&.info "Oj is not installed, and failing back to JSON for json parser"
+          configure_json_parser(:json)
+        when :json then [JSON.method(:parse), JSON::ParserError]
```

Comment:
Thanks for this enhancement!

> when :json then [JSON.method(:parse), JSON::ParserError]

This change means that `JSON#parse` is faster than `JSON#load`, right?
If so, could you please separate the PR?